ckanext/restricted/tests/test_plugin.py

```diff
@@ -273,3 +273,39 @@ def test_user_can_see_only_accessible_resource_in_package_search(self):
         assert package['num_resources'] == 1
         assert len(package['resources']) == 1
         assert package['resources'][0]['id'] == other_resource['id']
+
+    def test_org_member_see_resources_package_search(self):
+        """
+        A bug was flagged where the hide_inaccessible_resources param in
+        the package_search action was causing resources only to be visible to
+        organisation "editors" and "admins" .  Organisation "members" with read
+        but not write privileges could not see the resources in a
+        package_search action.  This test checks that org members can see their
+        resources in a package_search action with
+        hide_inaccessible_resources=True.
+        """
+        user = factories.User()
+        org = factories.Organization(
+            users=[{'name': user['id'], 'capacity': 'member'}]
+        )
+        restricted = '{{"level": "restricted", "allowed_organisations":[], "allowed_users":[]}}'
+        dataset = factories.Dataset(owner_org=org['id'], private=False)
+        resource = factories.Resource(
+            package_id=dataset['id'],
+            name="resource",
```

Comment:
you can remove the `name` kwarg from here since it's not used later in the test. It'll be autogenerated by the factory.